python/grass/jupyter/display.py

```diff
@@ -146,6 +148,7 @@ def run(self, module, **kwargs):
         # Check module is from display library then run
         if module[0] == "d":
             self._region_manager.set_region_from_command(module, **kwargs)
+            self._region_manager.adjust_rendering_size_from_region()
```

Comment:
If `use_region=False` in `GrassRenderer`, does this still set the extent from region anyways? That seems like it would be unexpected behavior but I'm having trouble telling if this is the case. And if not, since `adjust_rendering_size_from_region` only sets the variables the first time it's called (because of the `self._size_set` boolean), will the extent be set from which ever layer was added first? I'm also not sure that would be expected.

Reply:
It's using the same region rendering would use, so I don't see a problem, but actual testing should show that.

Reply:
My question is is if `adjust_rendering_size_from_region()` call should be external call like this or should be handled internally, but that's a low priority task for the API PR.